src/sentry/models/groupopenperiod.py

```diff
@@ -195,11 +217,31 @@ def get_open_periods_for_group(
     return open_periods
 
 
+def create_open_period(group: Group, start_time: datetime) -> None:
+    if not features.has("organizations:issue-open-periods", group.project.organization):
```

Comment:
nice! i like how this is merging all the info around the feature flag and nesting it into a single method. 🎉